pom.xml

```diff
@@ -22,7 +22,7 @@ under the License.
 
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.apache.reef</groupId>
-    <version>0.10-incubating-SNAPSHOT</version>
```

Comment:
Do we need to change the version for all pom.xml files (e.g. ./reef-common/pom.xml), or just this top-most one?